### PR TITLE
Make menu sticky and fix some css issue

### PR DESCRIPTION
**Pull request by ben181231**
connect #25
connect #26
connect #29

### CODE REVIEW

File: components/Guide/Guide.js [hunk 1]
```diff
@@ -1,10 +1,25 @@
 import React, { Component, PropTypes } from 'react';
+import classNames from 'classnames';
+
 import Markdown from '../../components/Markdown';
 import Menu from './Menu';
 
 import './Guide.scss';
+import { Window } from '../../lib/Location';
```
Comment (rickmak) [inline components/Guide/Guide.js]: This look wrong semantic to me.

File: components/Guide/Guide.js [hunk 2]
```diff
@@ -25,24 +40,55 @@ class Guide extends Component {
...
+  handleScroll() {
+    const scrollTop = Window.scrollY;
+    this.setState({
+      sticky: scrollTop > 230
+    });
+  }
+
+  componentDidMount() {
+    if (Window) {
+      if (Window.addEventListener) {
```
Comment (rickmak) [inline components/Guide/Guide.js]: I think it exist for all kind of window?

**Comment by rickmak** [comment]
I think clicking the link and skip back to page top seem not good. Is it difficult to add a scroll animation?


https://oursky.slack.com/files/rickmak/F14MXBZ99/untitled.mov

File: components/Guide/Guide.js (hunk 1 above)
Comment (ben181231) [inline components/Guide/Guide.js]: Rename the `Location.js` ?

File: components/Guide/Guide.js (hunk 2 above)
Comment (ben181231) [inline components/Guide/Guide.js]: No need to take care of IE ? XD

File: components/Guide/Guide.js (hunk 1 above)
Comment (rickmak) [inline components/Guide/Guide.js]: Yes. Can rename Location.js

Of wrap raw eventListener to a package. That package will do no-op on `canUseDom` is false.

**Comment by ben181231** [comment]
@rickmak PR updated.

**Pull request merged**